apps/openchallenges/mcp-server/src/main/java/org/sagebionetworks/openchallenges/mcp/server/config/McpServerConfigData.java

```diff
@@ -6,9 +6,27 @@
 
 @Data
 @Configuration
-@ConfigurationProperties(prefix = "openchallenges-mcp-server")
+@ConfigurationProperties
```

Comment:
Add prefix (see comments below)

Reply:
FYI, if we add the prefix, we need to add `OPENCHALLENGES-MCP-SERVER` to the `.env` as well, or we use the Kebab case - [dea4aee](https://github.com/Sage-Bionetworks/sage-monorepo/pull/3180/commits/dea4aee15d2c706c7d4270717e2615ab9f2d2c78)

Reply:
Good point! It's a bit verbose but this approach has the benefit to be descriptive when settings env vars for different dependencies (OPENCHALLENGES_MCP_SERVER_XXX, SPRING_DATA_YYY). Let's use the prefix because that's what other dependencies do like Spring.